app/views/modules/_access_control.html.erb

```diff
@@ -64,31 +64,33 @@ Unless required by applicable law or agreed to in writing, software distributed
   </div>
 </div>
 
-<div class="card item-lending-period">
-  <div class="card-header">
-    <h3 class="card-title">Item lending period</h3>
-  </div>
-  <div class="card-body">
-    <div class="form-group">
-      <%=  render partial: "modules/tooltip", locals: { form: vid, field: :lending_period, tooltip: t("access_control.#{:lending_period}"), options: {display_label: (t("access_control.#{:lending_period}label")+'*').html_safe} } %><br />
-      <% d, h = (lending_period/3600).divmod(24) %>
-      <div class='form-row'>
-        <div class="col-1">
-          <label class="col-form-label" for="add_lending_period_days">
-            <span data-title="Days">Days</span>
-          </label>
-          <%= text_field_tag "add_lending_period_days", d ? d : 0, class: 'form-control' %>
-        </div>
-        <div class="col-1">
-          <label class="col-form-label" for="add_lending_period_hours">
-            <span data-title="Hours">Hours</span>
-          </label>
-          <%= text_field_tag "add_lending_period_hours", h ? h : 0, class: 'form-control' %>
+<% if Settings.controlled_digital_lending.enable %>
+  <div class="card item-lending-period">
+    <div class="card-header">
+      <h3 class="card-title">Item lending period</h3>
+    </div>
+    <div class="card-body">
+      <div class="form-group">
+        <%=  render partial: "modules/tooltip", locals: { form: vid, field: :lending_period, tooltip: t("access_control.#{:lending_period}"), options: {display_label: (t("access_control.#{:lending_period}label")+'*').html_safe} } %><br />
+        <% d, h = (lending_period/3600).divmod(24) %>
+        <div class='form-row'>
+          <div class="col-1">
+            <label class="col-form-label" for="add_lending_period_days">
+              <span data-title="Days">Days</span>
+            </label>
+            <%= text_field_tag "add_lending_period_days", d ? d : 0, class: 'form-control' %>
+          </div>
+          <div class="col-1">
+            <label class="col-form-label" for="add_lending_period_hours">
+              <span data-title="Hours">Hours</span>
+            </label>
+            <%= text_field_tag "add_lending_period_hours", h ? h : 0, class: 'form-control' %>
+          </div>
         </div>
       </div>
     </div>
   </div>
```

Comment:
Could potentially pull this and the cdl bit at line 135 out into their own partial(s) in a future PR, though not sure it's worth it unless/until we need to render these settings outside of the access control context.

Reply:
:+1: I was thinking the same thing.